docker/Dockerfile-windows

```diff
@@ -1,6 +1,11 @@
 ARG WINDOWS_VERSION=1809
 #This version confirmed to work for both github and AKS
 FROM ghcr.io/dapr/windows-base:$WINDOWS_VERSION
+
+# Administrator access is required to install certificates.
+USER ContainerAdministrator
+
 ARG PKG_FILES
 WORKDIR /
+COPY /setup-certificates.cmd /
```

Comment:
Should this be added to the base dockerfile (`Dockerfile-windows-base`) instead? It would allow skipping the "awkward" copying here: https://github.com/dapr/dapr/pull/4741/files#diff-c5390b6b2b8f0bc8fb6702bb8aede7ad4d8344393456674aa8678480d56c4c76R93-R95